sponsorship_compassion/models/contracts.py

```diff
@@ -623,53 +623,8 @@ def suspend_contract(self):
                 message_type="comment",
             )
 
-        # Change invoices if config tells to do so.
-        if suspend_config:
-            product_id = int(suspend_config)
-            self._suspend_change_invoices(date_start, product_id)
-
         return True
 
-    @api.multi
-    def _suspend_change_invoices(self, since_date, product_id):
-        """ Change cancelled sponsorship invoices and put them for given
-        product. Re-open invoices. """
-        cancel_inv_lines = (
-            self.env["account.invoice.line"]
-                .with_context(lang="en_US")
-                .search(
-                [
-                    ("contract_id", "in", self.ids),
-                    ("state", "=", "cancel"),
-                    ("product_id.categ_name", "=", SPONSORSHIP_CATEGORY),
-                    ("due_date", ">=", since_date),
-                ]
-            )
-        )
-        invoices = cancel_inv_lines.mapped("invoice_id")
-        invoices.action_invoice_draft()
-        invoices.env.clear()
-        vals = self.get_suspend_invl_data(product_id)
-        cancel_inv_lines.write(vals)
-        invoices.action_invoice_open()
-
-    @api.multi
-    def get_suspend_invl_data(self, product_id):
-        """ Returns invoice_line data for a given product when center
-        is suspended. """
-
-        product = self.env["product.product"].browse(product_id)
-        vals = {
-            "product_id": product_id,
-            "account_id": product.property_account_income_id.id,
-            "name": "Replacement of sponsorship (fund-suspended)",
-        }
-        rec = self.env["account.analytic.default"].account_get(product.id)
-        if rec and rec.analytic_id:
-            vals["account_analytic_id"] = rec.analytic_id.id
-
-        return vals
-
     @api.multi
     def reactivate_contract(self):
```

Comment:
You can also remove completely method `def reactivate_contract` and the inheritance of method `def reactivate_project` in project_compassion.py